scenario/4_slice_named.yml

```diff
@@ -25,49 +25,70 @@ success: |
     type ConverterImpl struct{}
 
     func (c *ConverterImpl) Convert(source []execution.ID) []int {
-    	intList := make([]int, len(source))
+    	var intList []int
+    	if source != nil {
+    		intList = make([]int, len(source), len(source))
+    	}
     	for i := 0; i < len(source); i++ {
     		intList[i] = int(source[i])
     	}
     	return intList
     }
```

Comment:
I've changed the implementation to produce this:
```go
func (c *ConverterImpl) Convert(source []execution.ID) []int {
    var intList []int
    if source != nil {
        intList := make([]int, len(source))
        for i := 0; i < len(source); i++ {
            intList[i] = int(source[i])
        }
    }
    return intList
}
```
The groups puts the for loop into the nil check and imo looks a little better.

Furthermore, I've removed the capacity definition from the make call. If capacity is unsed the length will be used, so the old make call should be fine.